structcol/refractive_index.py

```diff
@@ -158,9 +159,202 @@ def n_silica_colloidal(w):
 def n_water(w):
     return 1.33
 
-def n_cargille(i,series,wavelength):
-    # where i is the cardinal number of the liquid (starting with 0) and w is the wavelength, in nm (converted to um)
-    return 1.45
+def n_cargille(i,series,w):
```

Comment:
need tests for these formulas (see note at top of file).  Also I'd prefer to find a way to add these to the dictionary above rather than defining as separate functions (see comments above).